src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
```

Comment:
This method should return color, in our program color is enum, so please return propriate value here